recipes-ofdpa/ofdpa/ofdpa_3.0.5.0-EA5.bb

```diff
@@ -4,10 +4,10 @@ LICENSE = "CLOSED"
 # Include SDK version, and OF-DPA and OpenBCM source revisions in version
 PV = "3.0.5.0-EA5+sdk-${SDK_VERSION}+gitAUTOINC+${@'${SRCREV_ofdpa}'[:10]}_${@'${SRCREV_sdk}'[:10]}"
 
-PR = "r7"
+PR = "r8"
 SDK_VERSION = "6.5.24"
 SRCREV_ofdpa = "a1b1e04908432e7ec42827febcd91ab9f9519127"
-SRCREV_sdk = "0b149ddfa3878e65eb217a11dddb999d3e205d03"
+SRCREV_sdk = "c2d7f9f3af8730338d4e7b8936e27991f94778a4"
```

Comment:
i can not find this commit in https://github.com/Broadcom-Network-Switching-Software/OpenBCM, what is referenced here?

Reply:
Good catch, I failed when updating the ofdpa commit id. Let me fix that.